runtime/shared_common/shrinit.cpp

```diff
@@ -3952,6 +3953,25 @@ j9shr_getCacheSizeBytes(J9JavaVM *vm)
 	return ret;
 }
 
+/**
+ * check if SCC is the default SSC (was enabled b/c of -Xshareclasses)
```

Comment:
Better to say "Check if the shared cache is enabled by default or using "-Xshareclasses" in the command line. "